### PR TITLE
Add article on managing ds records

### PR DESCRIPTION
**Pull request by lokst**
Belongs to dnsimple/dnsimple-business#1080

Document how to manage DS records

### CODE REVIEW

File: content/articles/manage-ds-record.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## Locating the DS records page
+
+To view and manage the delegation signer (DS) records for a domain, click on the DNSSEC tab on a domain's management page, then click on the <label>Manage</label> link in the DS Records section, to access the DS Records page.
```
Comment (ggalmazor) [inline content/articles/manage-ds-record.markdown:20]:
```suggestion
To view and manage the delegation signer (DS) records for a domain, click on the DNSSEC tab on a domain's management page, then click on the <label>Manage</label> link in the DS Records section.
```

File: content/articles/manage-ds-record.markdown [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+![Manage DS Records link](/files/ds-records-manage.png)
+
+## Listing DS records
+
+On the DS Records page, you will be able to see a list of the DS records currently configured for the domain.
```
Comment (ggalmazor) [inline content/articles/manage-ds-record.markdown:26]:
```suggestion
On the DS Records page, you will be able to see a list of the DS records currently provisioned for the domain.
```

File: content/articles/manage-ds-record.markdown [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+![List DS Records](/files/ds-records-list.png)
+
+## Adding a DS record
+
+On the DS records page, click <label>Add DS record</label>.
+
+![Add DS Record](/files/ds-records-add-ds-record.png)
+
+The Add DS Record page will load. The form fields shown on the Add DS record page will differ based on whether the TLD of the domain supports the DS-Data interface or the KEY-Data interface.
```
Comment (ggalmazor) [inline content/articles/manage-ds-record.markdown:36]:
```suggestion
The Add DS Record page will load. The form fields shown on the Add DS record page will differ based on whether the TLD of the domain requires them to be set up with the DS-Data interface or the KEY-Data interface.
```

Comment (lokst) [reply content/articles/manage-ds-record.markdown]: Great suggestion, thank you!

**Review by ggalmazor — APPROVED** [review]
Honestly, I think we could go with this! Great article :) Thanks!

**Review by OleMchls — COMMENTED** [review]
Great work :clap:

**Review by OleMchls — APPROVED** [review]

**Pull request merged**